_getting-started/concepts-system-overview-guide.md

```diff
@@ -0,0 +1,394 @@
+
+      - title: "Time to data loaded"
+        anchor: "replication-performance"
+        content: |
+          Because our process is performed in steps, it's important to note that Stitch replication isn't real-time. This means that there will be some time between data extracted and data loaded, which you can read more about in the [System architecture section](#system-architecture) of this guide.
+
+          Additionally, the speed of Extraction and Loading is largely dependent on the resources available in your data sources and destination.
+
+          > [TODO- do we have any recommendations for optimizing performance?]
+
+      - title: "Deleted records"
+        anchor: "replication--never-delete"
+        content: |
+          Stitch will never delete data from your destination, even if records have been hard deleted from the source. Refer to the [Deleted Record Handling guide]({{ link.replication.deleted-records | prepend: site.baseurl }}) for more info and examples.
+
+  - title: "Transformations"
+    anchor: "transformations"
+    summary: "The data transformations Stitch performs"
+    content: |
+      Stitch's goal is to get data from your data sources to your destination [in a useful, raw format](https://www.stitchdata.com/blog/why-our-etl-tool-doesnt-do-transformations/){:target="new"}:
+
+      > _Useful_ means with types and structures that make the data easy to work with, and _raw_ means staying as close to the original representation as possible.
+
+      This doesn't mean that Stitch doesn't perform transformations during the replication process. Stitch just doesn't perform _arbitrary_ transformations - Stitch will perform only the transformations required to ensure the loaded data is useful and compatible with your destination. The transformations Stitch performs are dependent on the destination you choose. These can include:
+
+      {% for subsection in section.subsections %}
+      - [{{ subsection.title }}](#{{ subsection.anchor }})
+      {% endfor %}
+
+      Stitch's philosophy is that what you do with your data depends on your needs, and by keeping data close to its original form, Stitch enables you to manage and transform it as you see fit. While we don't support user-defined transformations inside of Stitch, you can take advantage of [Talend's transformation and data quality solutions](https://www.stitchdata.com/platform/datatransformation/){:target="new"} to design and integrate your own transformations.
+
+    subsections:
+      - title: "Data typing"
+        anchor: "data-typing"
+        example-data:
+          - id: "1"
+            boolean: "true"
+            string: ""
+          - id: "2"
+            boolean: ""
+            string: "yes"
+          - id: "3"
+            boolean: ""
+            string: "no"
+        content: |
+          Stitch's data typing process consists of three steps which take place during replication:
+
+          1. **Extraction**: Identify the data type in the source
```

Comment:
since the bodies here are verbs (e.g. "Identify"), I think it would flow better to make the headers verbs too (e.g. "Extract")

Reply:
I think I changed this in the right places - check me:

- Data typing > Typing data
- JSON structures > Handling JSON structures
- Object names > Naming objects
- Timezones > Handling timezone data

Reply:
Oh I was actually just referring to this sublist:

`1. **Extraction**: Identify...` --> `1. **Extract**: Identify...`
`2. **Preparing**: Map...` --> `2. **Prepare**: Map...`
`3. **Loading**: Convert...` --> `3. **Load**: Convert...`

Reply:
lol oops. Got it!